composer.json

```diff
@@ -28,14 +28,21 @@
     "infinum/coding-standards-wp": "0.4.1",
     "dealerdirect/phpcodesniffer-composer-installer": "^0.6"
   },
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/infinum/eightshift-libs.git"
```

Comment:
You will change this once the latest version is released?

Reply:
Yes this is for testing and it is still in development so this will be changed later in the PR